epochs/hippieeg/config/snakebids.yml

```diff
@@ -0,0 +1,111 @@
+bids_dir: '/home/mcesped/projects/ctb-akhanf/cfmm-bids/Khan/epi_iEEG/ieeg'
+# bids_dir_tsv: '/home/mcesped/projects/ctb-akhanf/cfmm-bids/Khan/clinical_imaging/epi_iEEG/derivatives/seega_coordinates'
+output_dir: '/scratch/mcesped/epochs/results'
+
+#enable printing debug statements during parsing -- disable if generating dag visualization
+debug: False
+
+derivatives: False #will search in bids/derivatives if True; can also be path(s) to derivatives datasets
+
+#list of analysis levels in the bids app
+analysis_levels: &analysis_levels
+ - participant
+
+
+
+#mapping from analysis_level to set of target rules or files
+targets_by_analysis_level:
+  participant:
+    - ''  # if '', then the first rule is run
+
+#this configures the pybids grabber - create an entry for each type of input you want to grab
+# indexed by name of input
+#   dictionary for each input is passed directly to pybids get()
+#    https://bids-standard.github.io/pybids/generated/bids.layout.BIDSLayout.html#bids.layout.BIDSLayout.get
+
+pybids_inputs:
+  bold:
+    filters:
+      suffix: 'ieeg'
+      extension: '.edf'
+      datatype: 'ieeg'
+    wildcards:
+      - subject
+      - session
+      - acquisition
+      - task
+      - run
+
+pybids_inputs_tsv:
+  bold:
+    filters:
+      suffix: 'native_SEEG'
+      extension: '.tsv'
+    wildcards:
+      - subject
+      - session
+      - acquisition
+      - task
+      - run
+    custom_path: '/home/mcesped/projects/ctb-akhanf/cfmm-bids/Khan/clinical_imaging/epi_iEEG/derivatives/seega_coordinates/sub-P{subject}/sub-P{subject}_space-native_SEEGA.tsv'
```

Comment:
The folder was just updated by Greydon, so it should work now with the normal pybids structure. Will change it in the next commit!